pom.xml

```diff
@@ -56,14 +57,16 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>job-dsl</artifactId>
-            <version>1.84</version>
             <optional>true</optional>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>branch-api</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
This is already a transitive dependency from `branch-api` but better to define a direct dependency since we directly use the `symbol-arrow-forward-outline plugin-ionicons-api` symbol